### PR TITLE
First itteration of pca9685 implementation

### PR DESCRIPTION
**Pull request by Slimmerik**

### CODE REVIEW

File: code/headers/pca9685.hpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#include <hwlib.hpp>
+#include <i2c_bus.hpp>
+
+#pragma once
```
Comment (bartbal) [inline code/headers/pca9685.hpp]: #pragma once should be on the first line:

Rule: every header file starts with #pragma once on the first line. This replaces include guards.

File: code/src/pca9685.cpp [hunk 2]
```diff
@@ -0,0 +1,78 @@
+#include <pca9685.hpp>
+
+
+r2d2::pca9685::pca9685_c::pca9685_c(r2d2::i2c::i2c_bus_c &i2c_bus) : i2c_bus(i2c_bus){
```
Comment (bartbal) [inline code/src/pca9685.cpp]: initializer list should be on the next line.

Rule: short initializer lists are put on a new line:

File: code/headers/pca9685.hpp (hunk 1 above)
Comment (Slimmerik) [reply code/headers/pca9685.hpp]: done.

File: code/src/pca9685.cpp (hunk 2 above)
Comment (Slimmerik) [reply code/src/pca9685.cpp]: done.

**Pull request merged**